scripts/commands/setfamelevel.lua

```diff
@@ -0,0 +1,69 @@
+-----------------------------------
+-- func: setfamelevel
+-- desc: Sets fame level on a target player
+-----------------------------------
+local commandObj = {}
+
+commandObj.cmdprops =
+{
+    permission = 3,
+    parameters = 'iis'
+}
+
+local function error(player, msg)
+    if msg == nil then
+        msg = '!setfamelevel <fame_zone 0-15> <level 1-9> <player> (Omit level and target to show zone numbers)'
+    end
+
+    player:printToPlayer(msg)
+end
+
+commandObj.onTrigger = function(player, famezone, level, target)
+    -- validate target
+    local targ
+
+    if target == nil then
+        targ = player
+    else
+        targ = GetPlayerByName(target)
+        if targ == nil then
+            error(player, string.format('Player named "%s" not found or not a valid player!', target))
+            return
+        end
+    end
+
+    local fameZoneNames = {}
+    for name, value in pairs(xi.quest.fame_area) do
```

Comment:
I tried using this command while testing out #6216 and discovered that `xi.quest.fame_area` does not exist on LSB and consequently the command runs into an error.

Reply:
This PR preceeded the `xi.fameArea` change, I haven't made any updates as I don't have confirmation on the `fameBaseValues`. This was migrating code from ASB work that Critical did.

Reply:
I think I wrote this one originally, but yeah it was a while ago.  I can update if you want?

Reply:
Oh whoops, sorry @paladindamarus ! I had assumed from this PR: https://github.com/AirSkyBoat/AirSkyBoat/commit/00d8f060aa2e7284264faf9be01f8dc31c4e209c

Reply:
Will do... And no worries, I think it was either a bulk push or maybe a completely new update when she pushed.

Reply:
Any movement on this?

Reply:
Last bit I heard on this is the values were taken from Topaz.

Reply:
@Xaver-DaRed The base values match the values in lua_baseentity.cpp, `getFameLevel`, if that confirmation is all that's holding up this PR then we should be good to go unless we want to centralize these values somewhere.

Reply:
```suggestion
    for name, value in pairs(xi.fameArea) do
```